packages/scandipwa/src/route/Checkout/Checkout.container.js

```diff
@@ -501,10 +501,15 @@ export class CheckoutContainer extends PureComponent {
 
     handleRedirectIfLessThanMinAmountInCart() {
         const {
-            minimumOrderAmount: { minimum_order_amount_reached = true }
+            minimumOrderAmount: { minimum_order_amount_reached = true },
+            match: {
+                params: {
+                    step: urlStep = ''
+                }
+            }
         } = this.props;
 
-        if (!minimum_order_amount_reached) {
+        if (!minimum_order_amount_reached && !urlStep.includes(DETAILS_URL_STEP)) {
```

Comment:
I believe you have the `currentStep` prop available in this method.
Please use it instead of checking the substring of the url.

Reply:
We have checkoutStep state, I used it